From 72adac076e0d98fc2c3e846de72edf6169707cec Mon Sep 17 00:00:00 2001
From: Romain Perier <romain.perier@gmail.com>
Date: Thu, 1 Dec 2016 16:32:13 +0100
Subject: [PATCH] arm: mach-rk3188: Fix relative path for dependencies

Don't include relative path for dependencies modules, it breaks
out of source builds.

Signed-off-by: Romain Perier <romain.perier@gmail.com>
---
 arch/arm/mach-rk3188/Makefile | 24 ++++++++++++------------
 1 file changed, 12 insertions(+), 12 deletions(-)

diff --git a/arch/arm/mach-rk3188/Makefile b/arch/arm/mach-rk3188/Makefile
index b80abdc..2fb39fb 100644
--- a/arch/arm/mach-rk3188/Makefile
+++ b/arch/arm/mach-rk3188/Makefile
@@ -1,28 +1,28 @@
 EXTRA_CFLAGS += -Os
 ifneq ($(CONFIG_RK_FPGA),y)
-obj-y += ../plat-rk/clock.o
+obj-y += clock.o
 obj-y += clock_data.o
-obj-y += ../mach-rk30/ddr.o
-obj-y += ../mach-rk30/pmu.o
-obj-y += ../mach-rk30/reset.o
-obj-$(CONFIG_PM) += ../mach-rk30/pm.o
+obj-y += ddr.o
+obj-y += pmu.o
+obj-y += reset.o
+obj-$(CONFIG_PM) += pm.o
 endif
-obj-y += ../mach-rk30/common.o
+obj-y += common.o
 CFLAGS_common.o += -DTEXT_OFFSET=$(TEXT_OFFSET)
-obj-y += ../mach-rk30/devices.o
+obj-y += devices.o
 obj-y += io.o
 obj-y += rk_timer.o
-obj-$(CONFIG_SMP) += ../mach-rk30/platsmp.o ../mach-rk30/headsmp.o
-obj-$(CONFIG_HOTPLUG_CPU) += ../mach-rk30/hotplug.o
-obj-$(CONFIG_CPU_IDLE) += ../mach-rk30/cpuidle.o
+obj-$(CONFIG_SMP) += platsmp.o headsmp.o
+obj-$(CONFIG_HOTPLUG_CPU) += hotplug.o
+obj-$(CONFIG_CPU_IDLE) += cpuidle.o
 obj-$(CONFIG_CPU_FREQ) += cpufreq.o
 obj-$(CONFIG_DVFS) += dvfs.o
 obj-y += board.o
 
 board-$(CONFIG_MACH_RK3188_FPGA) += board-rk3188-fpga.o
-board-$(CONFIG_MACH_RK3188_TB) += ../mach-rk30/board-rk3168-tb.o
+board-$(CONFIG_MACH_RK3188_TB) += board-rk3168-tb.o
 board-$(CONFIG_MACH_RK3188_SDK) += board-rk3188-sdk.o
-board-$(CONFIG_MACH_RK3188_LR097) += ../mach-rk30/board-rk3168-LR097.o
+board-$(CONFIG_MACH_RK3188_LR097) += board-rk3168-LR097.o
 board-$(CONFIG_MACH_RK3188_DS1006H) += board-rk3188-ds1006h.o
 obj-$(CONFIG_SOC_RK3188) += board.o
 board-$(CONFIG_MACH_RK3188_FAC) += board-rk3188-fac.o
-- 
2.9.3

